### PR TITLE
Create db structure diagram

### PR DESCRIPTION
**Pull request by vanarman**
Created a DB diagram containing the following entities:
- Author
- Publisher
- Genre
- Book
- Format (hardcover, paper, ebook)
- User
- Cart
- Order

### CODE REVIEW

File: src/test/java/jpa/AuthorTest.java
Comment (YoussefSaghbini) [inline src/test/java/jpa/AuthorTest.java:1]: This file is already in the master, this branch may need to be updated before approval. As there might be conflict with #53 since that changes some method names.

File: src/test/java/jpa/PublisherTest.java
Comment (YoussefSaghbini) [inline src/test/java/jpa/PublisherTest.java:1]: Update branch as this file is also in the master branch, so no conflicts will occur when pushing.

**Review by YoussefSaghbini — CHANGES_REQUESTED** [review]
Please review comments.

**Review by YoussefSaghbini — APPROVED** [review]
Beautiful DB Diagram

**Review by MuneebNasir — APPROVED** [review]
Detailed Schema Looks Good

**Comment by MuneebNasir** [comment]
Merging on behalf of @vanarman

**Pull request merged**
